pom.xml

```diff
@@ -289,7 +298,7 @@
         <inbound.auth.oauth.version>6.2.0</inbound.auth.oauth.version>
         <commons-collections.version>3.2.0.wso2v1</commons-collections.version>
         <carbon.kernel.version>4.5.0-m3</carbon.kernel.version>
-        <identity.framework.version>5.14.0</identity.framework.version>
+        <identity.framework.version>5.14.46-SNAPSHOT</identity.framework.version>
```

Comment:
Need to change after framework merged

Reply:
In that case better to mention about the dependency that in the PR description